docs/flatten.rst

```diff
@@ -126,10 +126,12 @@ The field specified must be a field directly on the data object - it's not possi
 Remove Empty Schema Columns
 ---------------------------
 
-By default, columns that are empty (have no data) will be kept in the output.
+By default, when using this with `schema` specified columns that are empty (have no data) will be kept in the output.
 
 But you can pass the `remove-empty-schema-columns` flag to have these removed.
 
+If all columns are removed from a sheet and it is empty, the whole sheet will be removed to.
+
```

Comment:
Typo at the end? should it not be 'too'?
I would personally delete the 'too'. 'the whole sheet will be removed.'